prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -53,22 +55,21 @@ def omen_replicate_from_tx(
     api_keys: APIKeys,
     market_type: MarketType,
     n_to_replicate: int,
-    initial_funds: USD,
+    initial_funds: USD | CollateralToken,
```

Comment:
We also allow the `initial_funds` quantity to be specified on a per-settings way, which is given in collateral units.